src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -32,6 +33,7 @@ public class VodqaScreenAndroid extends VodqaScreen {
     private final By byWebViewSectionOptionXpath = AppiumBy.xpath("//android.view.ViewGroup[@content-desc='webView']");
     private final By byNativeViewSectionXpath = AppiumBy.xpath("//android.view.ViewGroup[@content-desc='chainedView']");
     private final String languageTextView = "//android.widget.TextView[@text=' %s']";
+    private final By byDragAndDropTextView = AppiumBy.xpath("//android.widget.TextView[@content-desc=\"dragAndDrop\"]");
```

Comment:
this can be [@content-desc='dragAndDrop'] instead of [@content-desc=\"dragAndDrop\"]